--- freetype-py-0.4.1/setup.py.orig	2012-12-31 09:08:23.000000000 +0100
+++ freetype-py-0.4.1/setup.py	2014-03-10 12:56:33.690024556 +0100
@@ -12,7 +12,7 @@
        author_email= 'Nicolas.Rougier@inria.fr',
        url         = 'http://code.google.com/p/freetype-py/',
        packages    = ['freetype', 'freetype.ft_enums'],
-       data_files  = ['examples/Vera.ttf',],
+       data_files  = [('share/freetype-py',['examples/Vera.ttf',])],
        classifiers = [
           'Development Status :: 5 - Production/Stable',
           'Environment :: X11 Applications',
